build.sbt

```diff
@@ -74,6 +82,7 @@ lazy val sdkJava = project
     // We need to generate the java files for things like entity_key.proto so that downstream libraries can use them
     // without needing to generate them themselves
     Compile / PB.targets += PB.gens.java -> crossTarget.value / "akka-grpc" / "main",
+    Compile / PB.protoSources ++= (codegenProtoAnnotations / Compile / PB.protoSources).value,
```

Comment:
Note that there are two things here. 

First, `codegenProtoAnnotations` is added as a dependency. We want users to get the proto file in the classpath when the pick the SDK jar so they can use them in their proto files. 
That could be an explicit dependency, but I think it should be included as a transitive deps of the SDK. The reason is that if an user update their project to the new SDK version, they will get the annotations and are good to go. Otherwise they would need to update their pom.xml. I think we should avoid it.

The SDK ships with the generated classes from the akkaserverless protocol, so here I'm doing the same. I let the SDK generated the new codegen proto classes and that then get included in the SDK jar.

Reply:
> The reason is that if an user update their project to the new SDK version, they will get the annotations and are good to go. Otherwise they would need to update their pom.xml. I think we should avoid it.

I agree, no extra dependency should be needed.